### ISSUE TITLE
moduleDirectory config is ignored

### ISSUE THREAD

**Issue by elliothursh** [smeijer/unimported#56]
The moduleDirectory config option is ignored when passed in through the unimportedrc.json file.

.unimportedrc.json
```
{
  "entry": ["application/src/index.js"],
  "extensions": [ '.ts', '.js', '.tsx', '.jsx' ],
  "ignorePatterns": [
      '**/node_modules/**',
      '**/__tests__/**'
  ],
  "ignoreUnresolved": [],
  "ignoreUnimported": [],
  "ignoreUnused": [],
  "moduleDirectory": ["node_modules", "application/src"]
}
```

```
❯ npx unimported --no-cache --show-config

npx: installed 215 in 8.162s
{
  version: '1.19.0',
  preset: 'node',
  flow: false,
  rootDir: undefined,
  ignoreUnresolved: [],
  ignoreUnimported: [],
  ignoreUnused: [],
  ignorePatterns: [
    '**/node_modules/**',
    '**/__tests__/**'
  ],
  entryFiles: [
    {
      file: 'application/src/index.js',
      aliases: {},
      extensions: [ '.ts', '.js', '.tsx', '.jsx' ]
    }
  ],
  extensions: [ '.ts', '.js', '.tsx', '.jsx' ]
}
```

**Comment by grit96** [comment]
This appears to be caused by a change between v1.17.0 and v1.18.0.

I have `"moduleDirectory": ["node_modules", "."]` in my config. Running `npx unimported@1.17.0` I get 19 unimported files, and running `npx unimported@1.18.0` I get 296 unimported files.

**Comment by smeijer** [comment]
I've published 1.19.1, which should fix this. Please let me know if the issue persists.

You might need to clean your caches with `npx unimported --clear-cache` or `npx unimported --no-cache`. See `--help` for more info about that.

**Comment by grit96** [comment]
Perfect! Works in 1.19.1
